app/poi_api/poi.py

```diff
@@ -3,52 +3,36 @@
 from flask import jsonify
 
 
-class POI(common.FirebaseDataEntity):
+class POI:

```

Comment:
There are a few unimplemented abstract methods from common.FirebaseDataEntity. This won't work until those are implemented